I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scripts/swipe.rpy", line 37, in script call
    call tips.give
  File "game/scripts/tips.rpy", line 112, in script call
    call tips.pick
  File "game/scripts/tips.rpy", line 121, in script
    python:
  File "game/scripts/tips.rpy", line 134, in <module>
    tip_title = tip["title"].lower()
KeyError: 'title'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scripts/swipe.rpy", line 37, in script call
    call tips.give
  File "game/scripts/tips.rpy", line 112, in script call
    call tips.pick
  File "game/scripts/tips.rpy", line 121, in script
    python:
  File "/Users/aljo/Downloads/5y1s/psych 180/renpy-8.2.0-sdk/renpy/ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/aljo/Downloads/5y1s/psych 180/renpy-8.2.0-sdk/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/scripts/tips.rpy", line 134, in <module>
    tip_title = tip["title"].lower()
KeyError: 'title'

macOS-15.0.1-arm64-arm-64bit arm64
Ren'Py 8.2.0.24012702
Simmer 1.0
Mon Nov  4 10:07:13 2024
